src/Metadata.elm

```diff
@@ -106,8 +119,10 @@ storyMetadataFromSlug slug contentDict =
         Just content ->
             Just
                 { title = content.title
-                , description = descriptionFromBody content.fullTextMarkdown
+                , description = content.summary
```

Comment:
I think this should be content.summary if there is one and the Key + text if not.